WalletWasabi.Gui/Tabs/WalletManager/RecoverWallets/RecoverWalletViewModel.cs

```diff
@@ -221,5 +210,33 @@ public void OnAddWord(string word)
 
 			Suggestions.Clear();
 		}
+
+		private void ValidateMinGapLimit(IValidationErrors errors)
+		{
+			if (!int.TryParse(MinGapLimit, out int minGapLimit))
+			{
+				errors.Add(ErrorSeverity.Error, "Must be a number.");
+			}
+			else if (minGapLimit < KeyManager.AbsoluteMinGapLimit)
+			{
+				errors.Add(ErrorSeverity.Error, $"Cannot be smaller than {KeyManager.AbsoluteMinGapLimit}.");
+			}
+			else if (minGapLimit > 1_000_000)
```

Comment:
As @lontivero suggested in https://github.com/zkSNACKs/WalletWasabi/pull/4000#discussion_r457468754 we should lower this 1 million number.
10_000 maybe?

Reply:
ACK